### DISCUSSION TITLE
Statics not always loading.

### DISCUSSION THREAD

**Discussion by buschNT** [unikubehq/cli#149] [General]
After switching an app using `unikube app switch` (the project was created using https://github.com/Blueshoe/django-hurricane-template), I don't always get statics.

![Screenshot from 2021-10-04 10-43-23](https://user-images.githubusercontent.com/6874607/135821373-e0779fd5-a7a9-41f8-addb-2a83458c01e4.png)

I this related to unikube/telepresence or is this a django-hurricane related problem?

**Comment by Schille** [comment]
What is the `command` from your unikube file?

**Reply by buschNT** [reply]
The command is as follows:
```
command: python manage.py serve --port {port} --autoreload
```

This should be from django-hurricane, or?

**Reply by Schille** [reply]
Yes. It this case django-hurricane does not serve the statics (even for `DEBUG=True`).
The `--static` flag is missing (hurricane does not service static requests at all) and I assume that you need to collect statics beforehand with `--command "collectstatic --no-input"` since Hurricane (in contrast to the django development server) does not automatically serve static files during development. 

However, this issue is not related to the Unikube CLI.

**Reply by Schille** [reply]
Please refer to Hurricane's documentation: https://django-hurricane.readthedocs.io/en/latest/usage.html#application-server

**Reply by buschNT** [reply]
Thanks!

Now I get the statics :D . I used the following command as suggested ("--static" alone was not sufficient):
`command: python manage.py serve --command "collectstatic --no-input" --port {port} --autoreload --static`
